skype/skype-ps/skype/Grant-CsTeamsIPPhonePolicy.md

```diff
@@ -0,0 +1,192 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+Module Name: Skype for Business Online
+online version: https://docs.microsoft.com/powershell/module/skype/grant-csteamsipphonepolicy
+applicable: Skype for Business Online
+title: Grant-CSTeamsIPPhonePolicy
+author: tonywoodruff
+ms.author: anwoodru
+ms.reviewer: kponnus
+manager: sandrao
+schema: 2.0.0
+---
+
+# Grant-CsTeamsIPPhonePolicy
+
+## SYNOPSIS
+
+Use the Grant-CSTeamsIPPhonePolicy cmdlet to assign a set of Teams phone policies to a user account or group of user accounts. Teams phone policies determine the features that are available to users of Teams phones. For example, you might enable the hot desking feature for some users while disabling it for others.
+
+
+## SYNTAX
+
+```
+Grant-CsTeamsIPPhonePolicy [-PassThru] [-Confirm] [[-PolicyName] <XdsIdentity>] [[-Identity] <Object>] [-Global]

```

Comment:
```suggestion
Grant-CsTeamsIPPhonePolicy [-PassThru] [-Confirm] [[-PolicyName] <String>] [[-Identity] <XdsIdentity>] [-Global]

```

Reply:
This one is interesting. As technically the XdsIdentity is the Policy Name held in the CMS. I left the -Identity as <Object> as it's the user object.